frontend_readme.md

```diff
@@ -0,0 +1,57 @@
+# **Productivity Cube - frontend**
+
+## Description
+This is a frontend part of Time Management Supporting System. The whole idea is about helping user with making decisions on what he should put his focus at the moment. We create a small cube, which analyzes our day at work and then
+predict if the action taken during some part of the day would be efficient or will be interrupted
+by other factors.
+
+
+## Installing
+bash
+```
+install dependencies
+npm install / yarn
+
+serve with hot reload at localhost:8080
+npm run dev / yarn start
+
+# build for production with minification
+npm run build / yarn build
+
+# build for production and view the bundle analyzer report
+npm run build --report / yarn build --report
+```
+
+## Documentation
+* Node.Js and NPM  - You can download Node.Js and NPM from - https://nodejs.org/en/download/
+* Vue - The web framework used - https://vuejs.org/v2/guide/
+
+## Usage or Screenshots
+
+On the welcome screen just type your name. You don't need password because it's just a school project, kind of proof of work.
+
+*Tip, need to delete later* <br>
+*should describe a general flow of using the app - how to login, which API endpoint is used there, why you don’t need a password, how to report an event etc.*
+
+## How To
+

```

Comment:
Please add a link to the cube-arduino project and describe where from the data comes - API and/or the cube hardware